src/extension/background-script/connectors/alby.ts

```diff
@@ -259,9 +261,7 @@ export default class Alby implements Connector {
       authClient.on("tokenRefreshed", (token: Token) => {
         this._updateOAuthToken(token);
       });
-      // Currently the JS SDK guarantees request of a new refresh token is done synchronously.
```

Comment:
can you re-add the comments? any reason why you removed them?

Reply:
aah! accidently deleted while cleaning up other comments. resolved